test/jdk/javax/sound/sampled/Lines/GetLine.java

```diff
@@ -35,18 +34,6 @@
  */
 public class GetLine {
 
-    static boolean isSoundAccessDenied = false;
-    static {
-        SecurityManager securityManager = System.getSecurityManager();
-        if (securityManager != null) {
-            try {
-                securityManager.checkPermission(new AudioPermission("*"));
-            } catch (SecurityException e) {
-                isSoundAccessDenied = true;
-            }
-        }
-    }
-
```

Comment:
Minor:
These are on unchanged lines.
Since AudioSystem.getLine(infos[0]) no longer throws security exception, the SecurityException catch block can be removed.

        try {
            l = AudioSystem.getLine(infos[0]);
        } catch(SecurityException lue) {
            log.println("SecurityException");
            return STATUS_PASSED;
        }

Reply:
> Minor: These are on unchanged lines. Since AudioSystem.getLine(infos[0]) no longer throws security exception, the SecurityException catch block can be removed.
> 
> ```
>     try {
>         l = AudioSystem.getLine(infos[0]);
>     } catch(SecurityException lue) {
>         log.println("SecurityException");
>         return STATUS_PASSED;
>     }
> ```

Good catch. It was so far out of sight I missed that. I've removed it.